web/src/components/Container.tsx

```diff
@@ -4,5 +4,5 @@ import { Container as C, ContainerProps } from '@material-ui/core'
 export const Container: React.FunctionComponent<ContainerProps> = (
   props: ContainerProps
 ) => {
-  return <C maxWidth="md" {...props} />
+  return <C maxWidth="xl" {...props} />
```

Comment:
Hmm, I don't know changing the component's default property is the right way. You could set `maxWidth` property when using the component.